packages/language-server/test/documentation-spec.ts

```diff
@@ -89,5 +98,83 @@ describe("The @ui5-language-assistant/language-server <getNodeDocumentation> fun
       const result = getNodeDocumentation(ui5Enum, ui5SemanticModel);
       expect(result.value).to.include("Experimental.");
     });
+
+    it("get the UI5 node name with a model", async () => {
+      const xmlSnippet = `<mvc:View displayBlock="true"
+      xmlns="sap.m"
+      xmlns:pwd="openui5.password"
+      xmlns:mvc="sap.ui.core.mvc">
+      <Page>
+        <content>
+          <Input liveChange="onInput" />
+        </content>
+      </Page>
+    </mvc:View>`;
+      const { document, position } = getXmlSnippet(xmlSnippet);
+
+      const result = await getUI5NodeName(
+        document.offsetAt(position),
+        document.getText(),
+        ui5SemanticModel
+      );
+      if (result) {
+        expect(getNodeDetail(result)).to.equal("sap.m.Input");
+      } else {
+        expect(result).to.equal("sap.m.Input");
+      }
+    });
+
+    it("get the UI5 node name without a model", async () => {
+      const xmlSnippet = `<mvc:View displayBlock="true"
```

Comment:
these snippets normally need a "⇶" symbol to mark the cursor position.
Why is this not needed here?